src/components/form/HdRadio.vue

```diff
@@ -1,57 +1,53 @@
 <template>
+
   <section
-    :class="wrapperClasses"
-    class="radio-wrapper"
     @keydown="setUsingMouse(false)"
     @mousedown="setUsingMouse(true)"
   >
-    <label
-      v-if="label"
-      :id="`${name}--label`"
-      class="radio-wrapper__label"
-      tabindex="-1"
-    >{{ label }}</label>
     <div
-      :id="name"
-      :aria-labelledby="label ? `${name}--label` : null"
-      class="radio-wrapper__items"
       role="radiogroup"
+      class="radio-wrapper"
+      :class="{
+        'radio-wrapper--errored': hasError,
+        'radio-wrapper--disabled': disabled,
+        'radio-wrapper--vertical': vertical,
+        'radio-wrapper--using-mouse': isUsingMouse,
+      }"
     >
-      <!-- eslint-disable vue/valid-v-on -->
       <div
         v-for="(item, i) in items"
-        :key="`radio-${item.value}`"
+        :key="item.value"
         ref="items"
         :aria-checked="item.value === value ? 'true' : 'false'"
         :class="{
-          'isSelected': item.value === value,
+          'radio--selected': item.value === value,
         }"
         :tabindex="getTabindex(item, i)"
```

Comment:
tab-index always returns `-1` for elements.  fyi
<img width="704" alt="Screenshot 2020-09-29 at 10 23 49" src="https://user-images.githubusercontent.com/14016028/94532294-f6b13000-023d-11eb-8563-73dcace33a41.png">

Reply:
Thanks for the comment 😄 

And hmm I didn't change the [original behavior](https://github.com/homeday-de/homeday-blocks/pull/566/files/86ccd7e4d734fe13780bd0ab7bc4133523af643b#diff-e1c07f8f23d0f5161637f99697817ae0R129) or the component, so I don't know exactly why the component is working this way.

I think the `-1` is being used to enable the user to directly focus on the `selected` option.
But in this case, do you think I should remove that and set all tab indexes to 0?

Reply:
We shouldn't change anything if we use `tabindex` to directly focus on the `selected` option. All good!